content/YubiHSM2/Usage_Guides/YubiHSM_2_for_Microsoft_Host_Guardian_Service:_Deployment_Guide/Back_Up_Key_Material.adoc

```diff
@@ -0,0 +1,8 @@
+== Back Up Key Material
+
+Yubico strongly recommends making a backup copy of all production objects residing on your production devices, particularly once the HGS signing and encryption keys have been generated on the YubiHSM 2. If there is a hardware failure of the production device, having a backup ensures that you can resume operations quickly. The backup process will result in two identical YubiHSM 2 devices with the same number of objects, keys, labels, etc.
+
+Backup the YubiHSM 2 according to the instructions in link:../YubiHSM_2_Windows_Deployment_Guide--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server/Backup_and_Restore_Key_Material.adoc[the corresponding chapter of the YubiHSM 2 Windows Deployment Guide].
```

Comment:
asciidoc automatically converts the -- to — to avoid this it has to be written as \-- I'm not sure if this is worth remembering everywhere or if we should change the directory name to not have -- in it.

Reply:
Most of these guides have lengthy titles, with the subtitle coming after a colon. I noticed that the tables of contents on the left-hand side don't show the title+subtitle combination properly if a colon is used. My solution was changing the colons in the directory names to double hyphens. What would you suggest instead?

Reply:
I'm fine with the double hyphen, but the link to it breaks, asciidoc replaces the -- in the link with —, though the directory name still has --. I see that github swallows my escaping of it though, in the link it has to be \\-- to not be converted.

Reply:
For my part, I am fine with the long dash ( — ) in the titles; all I wanted was punctuation equivalent to a colon because the colon itself wouldn't display properly. In an attempt to comply with your change request by retaining the double hyphen (--), I am making a new commit to this PR with all applicable folder names changed, and links changed to match. Unfortunately, however, the git command line is showing me that replacing "--" with "\--" results in "\\--". If this isn't acceptable, please kill the new commit. I will need to tell Sebastian that he can't have long titles.